### DISCUSSION TITLE
“Arctic Code Vault Contributor” Badge is missing？

### DISCUSSION THREAD

**Discussion by andybuibui** [community/community#129685] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

1. I have changed my GitHub account name.
2. My email address remains unmodified.
3. After changing my account name, I also deleted all the projects.
The badge has disappeared.

**Comment by davevad93** [comment, marked as answer]
Hi @andybuibui ,

If you have deleted all your projects, that's probably why the `Arctic Code Vault Contributor` badge is gone. When you remove a repo, any related contributions are also erased from the contribution graph. Some badges depend on the repo that earned you the achievement, thus by deleting it the badge also disappears.
